tests/test_transforms/test_feature_selection/test_filter_transform.py

```diff
@@ -97,3 +97,105 @@ def test_exclude_filter_wrong_column(ts_with_features):
     transform = FilterFeaturesTransform(exclude=["non-existent-column"])
     with pytest.raises(ValueError, match="Features {.*} are not present in the dataset"):
         ts_with_features.fit_transform([transform])
+
+
+@pytest.mark.parametrize("return_features", [True, False])
+@pytest.mark.parametrize(
+    "columns, saved_columns",
+    [
+        ([], ["target", "exog_1", "exog_2"]),
+        (["target"], ["exog_1", "exog_2"]),
+        (["exog_1", "exog_2"], ["target"]),
+        (["target", "exog_1", "exog_2"], []),
+    ],
+)
+def test_transform_exclude_save_columns(ts_with_features, columns, saved_columns, return_features):
```

Comment:
In these tests you should check that transform correctly stores columns in _df_removed